frontend/src/components/SignupForm.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect } from 'react'
+import { batch, useDispatch, useSelector } from 'react-redux'
+import { useHistory } from 'react-router-dom'
+
+import { account } from '../reducers/account'
+
+import { API_URL, SIGNUP } from '../reusable/urls'
+
+const SignupForm = () => {
+  const [username, setUsername] = useState('')
+  const [email, setEmail] = useState('')
+  const [password, setPassword] = useState('')
+
+  const accessToken = useSelector(store => store.account.accessToken)
+  const dispatch = useDispatch()
+  const history = useHistory()
+
+  useEffect(() => {
+    if (accessToken) {
+        history.push('/authorized')
```

Comment:
This does not seem to work. When signing up you just stay on the same page and the user does not get any feedback.